Protection Profile/BiocPP.adoc

```diff
@@ -455,37 +457,11 @@ Target error rates defined in SFR shall be evaluated based on <<BIOSD>>. Normall
 
 ===== FIA_PAD_EXT.1 Presentation attack support [[FIA_PAD_EXT.1]]
 
-*FIA_PAD_EXT.1.1*:: The TSF shall provide the [*selection*: _biometric modality_] with [*selection*: _enrolment detection_] detection at biometric enrolment and [*selection*: _verification detection equal to or greater than enrolment detection_] detection at biometric verification.
+*FIA_PAD_EXT.1.1*:: The TSF shall provide the [*selection*: _eye, face, fingerprint, vein_] with [*selection*: _PAD-L3, PAD-L2, PAD-L1, no PAD_] detection as specified in FIA_MBE_EXT.3 during biometric enrolment and [*selection, equal or greater than enrolment*: _PAD-L3, PAD-L2, PAD-L1, no PAD_] detection as specified in FIA_MBV_EXT.3 during biometric verification.
```

Comment:
```suggestion
*FIA_PAD_EXT.1.1*:: The TSF shall provide the [*selection*: _eye, face, fingerprint, vein_] with [*selection*: _PAD-L3_ (if available), _PAD-L2_ (if available), _PAD-L1_, _no PAD_] detection as specified in FIA_MBE_EXT.3 during biometric enrolment and [*selection, equal or greater than enrolment*: _PAD-L3, PAD-L2, PAD-L1, no PAD_] detection as specified in FIA_MBV_EXT.3 during biometric verification.
```

If you're allowed to use conditionals in selection (and it appears you can because we do it in *FIA_MBV_EXT.3.1*), then you could do something like this. Then the note is still explanatory and not normative. I think this helps resolve the issue I'm foreseeing.

Reply:
I can see where you are going, but it if we need to do that, then we should probably put it as the selection, like

[**selection, from available PAD levels for the modality**: _PAD-L3, PAD-L2, PAD-L1, no PAD_]

I don't really like the "if available" as part of the selection itself.

Reply:
I would be fine with that. Let's perhaps get some other opinions on if this is necessary.

Reply:
First selection should be:

[**selection, from available PAD levels**: _PAD-L3, PAD-L2, PAD-L1, no PAD_]

Second selection should be:

[**selection, from available PAD levels, equal or greater than enrolment**: _PAD-L3, PAD-L2, PAD-L1, no PAD_]